examples/chatbot_llm/main.py

```diff
@@ -1,25 +1,152 @@
-import argparse
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Usage: main.py [OPTIONS] MODEL_PATH
+
+  Command Line utility to interact with a text genration LLM in a chatbot
+  style
+
+  Example usage:
+
+  python main.py [OPTIONS] <MODEL_PATH>
+
+Options:
+  -s, --session_id TEXT           Name of the session that will be used in the
+                                  example  [default: session_1]
+  -m, --max_generated_tokens INTEGER
+                                  Max number of tokens to generate  [default:
+                                  32]
+  --sequence_length INTEGER       Sequence length to compile model and
+                                  tokenizer for.This controls the maximum
+                                  context length of the pipeline.  [default:
+                                  512]
+  --sampling_temperature FLOAT    The temperature to use when samplingfrom the
+                                  probability distribution computed from the
+                                  logits.Higher values will result in more
+                                  random samples. Shouldbe greater than 0.0.
+                                  [default: 1.0]
+  --prompt_length INTEGER         Processed prompt in chunks of this length.
+                                  This is to maximize the inference speed
+                                  [default: 64]
+  --token_generation_speed / --no-token_generation_speed
```

Comment:
```suggestion
  --show_tokens_per_sec
```
shorter and more descriptive

Reply:
+1 on this :)